user_tools/src/spark_rapids_pytools/resources/databricks-azure-price-westus2-raw.txt

```diff
@@ -0,0 +1,195 @@
+DS3 v2	4	14.00 GiB	0.75	$0.225/hour	$0.454/hour	$0.349/hour ~23% savings	$0.306/hour ~33% savings	$0.249/hour ~45% savings
```

Comment:
The resources folder will be hard to browse as we add to it in a flat manner. Can we move those files to a different directory?
If they are supposed to generate static files that will be saved in "resources", then I suggest that we have a "dev" folder somewhere else for those sort of scripts that should not be part of the packaging since they are not gonna be used in runtime..
Unless you plan to run those scripts in runtime to update the resources.

IIUC, the python file processes the raw.txt file. Can we have more detailed description in the python file that explains the assumptions, and how to use the script.

Regarding the txt file, How was it generated?

Reply:
Thank you!
I moved the Python script and raw files into a "dev" folder under "resources".
I added description about how the raw.txt file was generated in the python file.